### ISSUE TITLE
Needs fixing for LogSeq 0.10.5

### ISSUE THREAD

**Issue by wrmbaron** [kirso/logseq-panic-theme#11]
In LogSeq 0.10.5, Panic is completely unusable. The background is white and lots of details have become invisible. Please update Panic, because I really like it!

**Comment by kirso** [comment]
> In LogSeq 0.10.5, Panic is completely unusable. The background is white and lots of details have become invisible. Please update Panic, because I really like it!

Thanks for notification, I wasn't maintaining it all for 2 years now. I think logseq changed quite a bit in the meantime. I'll try to look at it early Feb and make it usable again and sorry for bad UX as of now :(

**Comment by wrmbaron** [comment]
I noticed, that it might be version 0.10.4 or 0.10.5 of LogSeq, which is the problem. There are a number of new [LogSeq issues](https://github.com/logseq/logseq/issues) about colors and custom css.

**Comment by kirso** [comment]
> I noticed, that it might be version 0.10.4 or 0.10.5 of LogSeq, which is the problem. There are a number of new [LogSeq issues](https://github.com/logseq/logseq/issues) about colors and custom css.

Yep, it broke most of the themes on the marketplace quite frankly, also with a lot of weird contrast happening.

**Comment by kenphused** [comment]
As an FYI, the theme will work after setting an Accent color  ([Settings] -> [Accent color]).  Dark and light modes work as expected but may override some theme settings.
